lib/blockchain_test.go

```diff
@@ -762,14 +763,32 @@ func _signTxnWithDerivedKey(t *testing.T, txn *MsgDeSoTxn, privKeyStrArg string)
 	privKeyBytes, _, err := Base58CheckDecode(privKeyStrArg)
 	require.NoError(err)
 	privateKey, publicKey := btcec.PrivKeyFromBytes(btcec.S256(), privKeyBytes)
-	if txn.ExtraData == nil {
-		txn.ExtraData = make(map[string][]byte)
-	}
-	txn.ExtraData[DerivedPublicKey] = publicKey.SerializeCompressed()
-	txnSignature, err := txn.Sign(privateKey)
-	require.NoError(err)
 
-	txn.Signature = txnSignature
+	// We will randomly sign with the standard DER encoding + ExtraData, or with the DeSo-DER encoding.
+	signatureType := rand.Int() % 2
+	if signatureType == 0 {
```

Comment:
It's tricky. It would require modifying many tests, and I don't think it'll increase the coverage by too much. I think the majority of the issues with the signatures would come up immediately, and going with option 3. by default should capture them.

Reply:
That's fine by me then